### ISSUE TITLE
Enhancing the management and exposure of persistent universal identifiers

### ISSUE THREAD

**Issue by lsetiawan** [ODM2/CZ-Manager#35]
Hi @miguelcleon  and @emiliom,

I am currently, enhancing ODM2Admin by displaying IGSN in the Sampling Features change list, and allow one to enter IGSN when creating a Sampling Feature. I have also added a feature to display ORCID for a Person and their affiliation. Additionally, one can enter a person's ORCID within the person add form. 

I improved the UI a bit for listing editable item types in addition to those enhancements. 

You can follow my changes here: [link](https://github.com/lsetiawan/ODM2-Admin/tree/crossczo)

Here are some screenshots if you're interested: [link](https://drive.google.com/open?id=0B2fuZWlhXcqQdE5KREhtRTN1Qm8)

Thanks.

**Comment by miguelcleon** [comment]
Cool, I checked out the screen shoots, looks great. Do a pull request when your ready.

**Comment by emiliom** [comment]
Thanks, @lsetiawan. @miguelcleon, it'll probably be another week or more before we're ready to do a PR. But my goal is to have these new features in place to demonstrate from a local / laptop instance at the Montreal workshop.

**Comment by miguelcleon** [comment]
sounds good, l'm looking forward to giving it a try @lsetiawan @emiliom

**Comment by miguelcleon** [comment]
Hi, how is this coming along? @lsetiawan @emiliom

**Comment by lsetiawan** [comment]
Hi @miguelcleon, It is coming along great. I'm still doing few tweaks here and there to improve the UI.

**Comment by emiliom** [comment]
Yup. @lsetiawan is now exposing doi's, ORCID id's and IGSN's in a nice and user-friendly way, as links to the external records. I'll show this in Montreal next week. Don should be able to submit a PR next wee, once the code stabilizes and we're less focused on entering the cross-CZO sampling information (for the meeting), thinking through UI implications, etc.

**Comment by miguelcleon** [comment]
Thinking about doing a PR today? @lsetiawan

**Comment by emiliom** [comment]
@miguelcleon, I've asked @lsetiawan to hold off until after the workshop. Mostly, b/c I need his time! At the workshop, you and I can chat about the changes Don has made and discussions we've had; I can show them to you on my laptop. Then after coordinating with Don, he can do a PR after Wednesday. Sound good?

**Comment by miguelcleon** [comment]
ok

**Comment by miguelcleon** [comment]
@emiliom can @lsetiawan issue the PR now I'd really like to see those changes you should role into the site. Thanks!

**Comment by emiliom** [comment]
@lsetiawan was getting set to do that today, but we weren't able to touch base on this today. We'll discuss tomorrow (Tue), and hopefully he'll be able to follow up with a PR or more soon after.

In the meantime, if you're **really** curious, [all his changes are on his fork, on the `crossczo` branch](https://github.com/lsetiawan/ODM2-Admin/tree/crossczo). At least changes through last Wednesday, which include everything I told you about or showed at the workshop.

**Comment by miguelcleon** [comment]
Yeah I was thinking about that, I'd prefer a PR, showing the site again on Wednesday morning so I might pull from it but I don't want to if it's not ready.

**Comment by emiliom** [comment]
Got it. I have no clue how much extra work it'll be for Don to create the PR(s) after he and I talk tomorrow. But hopefully it won't be a big deal, and he'll be able to submit by your mid afternoon, maybe earlier.

**Comment by miguelcleon** [comment]
Hopefully shouldn't be a big deal. Thanks!

**Comment by emiliom** [comment]
Don and I discussed the PR an hour ago. He will submit it **very** soon, if he hasn't done so already. His availability to answer questions this week may be limited, but from what he tells me, the PR has few or no conflicts relative to your master branch, so hopefully you'll be able to merge easily.
